libcxx/utils/generate_feature_test_macro_components.py

```diff
@@ -2555,6 +2645,72 @@ def generate_header_test_directory(self, path: os.path) -> None:
                 f.write(self.generate_header_test_file(header))
 
 
+    @functools.cached_property
+    def status_list_table(self) -> str:
+        """Creates the rst status table using a list-table."""
+
+        result = ""
+        for std in self.std_dialects:
+            result += create_table_row(
+                [[f'**{std.replace("c++", "C++")}**', "", "", "", ""]]
+            )
+
+            for feature in self.__data:
+                if std not in feature["values"].keys():
+                    continue
+
+                row = list()
+
+                ftm = feature["name"]
+                libcxx_value = (
+                    f"{self.standard_ftms[ftm][std]}"
+                    if self.is_implemented(ftm, std)
+                    else "*unimplemented*"
+                )
+
+                values = feature["values"][std]
+                assert len(values) > 0, f"{feature['name']}[{std}] has no entries"
+                for value in values:
```

Comment:
```suggestion
                for value, papers in values.items():
```

Would that work too?

Reply:
no it does not work, I've tested it.